<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>License_Dialog</class>
 <widget class="QDialog" name="License_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>611</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>551</width>
    <height>611</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>551</width>
    <height>611</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Software license</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>To obtain a license KEY number you must first purchase a software license</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Have you purchased?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="_100">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>53</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="_101">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_103">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buy now</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Type of license</string>
   </property>
   <widget class="QRadioButton" name="_104">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>30</y>
      <width>309</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Normal stand-alone machine license                              </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="_105">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>50</y>
      <width>303</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Network server license                                                    </string>
    </property>
   </widget>
   <widget class="QPushButton" name="_106">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>26</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Network</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>531</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Online Key Activation</string>
   </property>
   <widget class="QPushButton" name="_113">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>76</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get license</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="_111">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>22</y>
      <width>395</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>This machine is connected to the internet and I have my serial number    </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="_114">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>48</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>127</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Email address</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Software serial number</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="_115">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>79</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>531</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Offline Key Activation</string>
   </property>
   <widget class="QLineEdit" name="_107">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>80</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>79</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Software LOCK#</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Step 2. Contact CAD International to obtain your activation key.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="_112">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>24</y>
      <width>303</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>I want to manually activate my license offline    </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Step 1. Copy the LOCK number and send to CAD International</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="_108">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>78</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>531</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="_110">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>60</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>10</y>
      <width>363</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Step 3. Type or Paste your supplied activation KEY below.....</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="_1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>58</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Validate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_109">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>58</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(A to F, digits 0 to 9 and - only)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>568</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Trial mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>570</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Exit to </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>_100</tabstop>
  <tabstop>_101</tabstop>
  <tabstop>_103</tabstop>
  <tabstop>_104</tabstop>
  <tabstop>_105</tabstop>
  <tabstop>_106</tabstop>
  <tabstop>_111</tabstop>
  <tabstop>_114</tabstop>
  <tabstop>_115</tabstop>
  <tabstop>_113</tabstop>
  <tabstop>_112</tabstop>
  <tabstop>_107</tabstop>
  <tabstop>_108</tabstop>
  <tabstop>_110</tabstop>
  <tabstop>_109</tabstop>
  <tabstop>_1</tabstop>
  <tabstop>_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
